manifests/s/Storj/Uplink/1.127.1/Storj.Uplink.locale.en-US.yaml

```
@@ -0,0 +1,108 @@
+  - e1b6839 satellite/overlay: rename GetActiveNodes
+  - 9215fa2 web/satellite: small update for edit default CC
+  - 2b29e89 satellite/satellitedb: drop table storagenode_bandwidth_rollups_phase2
+  - dea7686 satellite/jobq: Fix switching queues via Insert
+  - 285dad7 satellite/nodeselection: add stream implementation for node selection
+  - e86509f satellite/jobq: instrument with monkit
+  - bcef33a web/satellite: updates for session handling
+  - 0ce9356 satellite/console: Fix UserStatus String method
+  - cb78553 satellite/{console,web}: remove self-serve placement names config
+  - 99ef283 satellite/{admin,console}: set external_id to nil on user deactivation
+  - e51a930 satellite/admin: add functionality to update user's external ID
+  - f12ad03 satellite/repair/repairer: handle ctx in WaitForPending repairs
+  - 5fc4a5a satellite/jobq: switch to a minmax heap
+  - c8e0e4f cmd/satellite: Add new sub-command set status user accounts
+  - 77ed138 satellite/jobq: add a maximum size to queues
+  - 8646bad cmd/satellite: Bind the config to last added command
+  - f12b024 satellite/satellitedb: use Spanner Read API for GetBucket
+  - 7c67784 web/satellite: error msg fix for add funds
+  - 0cbd9a0 satellite/metabase: add stale reads and low priority to ListBucketsStreamIDs
+  - f6baa10 satellite/console: reset step counter on successful project delete
+  - cb515d4 satellite/metainfo/bloomrate: convert to token bucket design
+  - 533df0a satellite/audit: support audit of filtered nodes
+  - 07dd2ec satellite: experimental Dockerfile for the (experimental) modular satellite
+  - 212a4d4 satellite/metabase: remove deprecated db
+  - f47e1f9 satellite/overlay: rename method TestSetNodeCountryCode
+  - 94a5498 satellite/console: add config for STORJ zkSync contract address
+  - a660a86 web/satellite: improve storj deposit dialog
+  - 745fed8 satellite/metabase: add MaxCommitDelay option for object commit
+  - da27a56 satellite:{payments,web}: fallback to global placements
+  - 4ad554a satellite/metabase: don't set TrackSessionHandles for Spanner session pool
+  - 414322e satellite/jobq: use drpc connection pooling
+  - f4820e1 satellite/durability: avoid keeping alias map in memory
+  - a767e00 satellite/repair: add participating nodes cache
+  Storagenode
+  - 2cda209 storagenode/blobstore/statcache: remove cache entry on trash restore
+  - 28065d0 storagenode/hashstore: some helper function improvements
+  - f315534 storagenode/hashstore: mmap support
+  - 11c292d storagenode/hashstore: small bug fixes/improvements
+  - 47d9618 storagenode/hashstore: remove madvise
+  - 6a8be1a storagenode/hashstore: make ConfigTest os agnostic
+  - 8d462dd storagenode/hashstore: lazy+sem acquire log files
+  - 2968fa5 storagenode/hashstore: use constructors for new tables
+  - 17d97aa storagenode/hashstore: use custom map type
+ReleaseNotesUrl: https://github.com/storj/storj/releases/tag/v1.125.2
+ManifestType: defaultLocale
+ManifestVersion: 1.9.0
```

```suggestion
ManifestVersion: 1.10.0
```

In the installer manifest you use manifest version 1.10.0. in other manifests the manifest version number should be standardized to `1.10.0` instead of `1.9.0`.